.github/ISSUE_TEMPLATE.md

```diff
@@ -1,44 +0,0 @@
-<!--
-Please use the following template to assist with creating an issue and to ensure a speedy resolution. If an area is not applicable, feel free to delete the area or mark with `N/A`
--->
-
```

Comment:
I don't think we should be removing the legacy template. There's a good chance someone out there still holds the `https://github.com/clap-rs/clap/issues/new` link which leads to this template currently. Better to have at least some template than none at all.